napari/components/_viewer/model.py

```diff
@@ -244,8 +244,8 @@ def add_layer(self, layer):
         layer.events.name.connect(self._update_name)
 
         self.layers.append(layer)
+        layer.parent = self._view
```

Comment:
Do we really want this to be public?

Reply:
probably not, but this will hopefully get removed soon. is it ok to leave for now?

Reply:
it will get removed when we pull out the vispy stuff from each layer's model file, as mentioned bellow